examples/daal/cpp_sycl/makefile_lnx

```diff
@@ -104,7 +104,7 @@ LOPTS := -Wl,--start-group -Wl,$(DAAL_LIB),$(EXT_LIB) $(DAAL_PATH)/libonedal_syc
 RES_DIR=_results/$(compiler)_$(_IA)_$(threading)_$(RES_EXT)
 RES = $(addprefix $(RES_DIR)/, $(if $(filter run, $(mode)), $(addsuffix .res ,$(example)), $(addsuffix .exe,$(example))))
 
-ifeq ($(compiler),clang)
```

Comment:
Let's ensure that clang still will work - we shouldn't break make compiler=clang case at least for now. Then later we can deprecate it.